piglet/src/test/java/org/apache/calcite/test/PigRelExTest.java

```diff
@@ -67,11 +68,13 @@ private void checkType(String pigExpr, Matcher<String> rowTypeMatcher) {
 
   @Test
   public void testConstantBoolean() {
+    Assume.assumeFalse("Skip: Pig/Hadoop tests do not work on Windows", isWindows());
```

Comment:
Instead of repeating the `assumeFalse` call in every method I guess you would achieve the same by using `@Before` or `@BeforeClass` annotation.

Reply:
Yep, will do

Reply:
Refactored